packages/kit/src/views/Developer/pages/Gallery/Components/stories/CryptoGallery.tsx

```
@@ -0,0 +1,715 @@
+function SecretFunctionsTest() {
+  const [result, setResult] = useState('');
+
+  const testSecretFunctions = async () => {
+    try {
+      const tasks: IRunAppCryptoTestTaskResult[] = [];
+
+      const testPasswordRaw = 'password123';
+      const encodeSensitiveTextKey =
+        'ENCODE_KEY::755174C1-6480-401A-8C3D-84ADB2E0C376::cf6e2e1c-e53b-431e-a6e4-3f27c9a7ac0b';
+
+      let customSalt = bufferUtils.toBuffer(
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'customSalt' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)